src/validations/vehicleManagerValidations.ts

```diff
@@ -0,0 +1,25 @@
+import { ValidationFunction } from '../hooks/useErrorHandler';
+import { getParamFromUrlOrState } from '../utils/urlHelpers';
+
+const validatePermissionTemplate: ValidationFunction = ({ urlParams, decodedState }) => {
+  const permissionTemplateId = getParamFromUrlOrState(
+    'permissionTemplateId',
+    urlParams,
+    decodedState,
+  );
+
+  if (!permissionTemplateId) {
```

Comment:
Would this conflict with the new flexible `permissions` system you have going on with https://linear.app/dimo/issue/ACC-546/liwd-support-more-permission-templates-in-a-systematic-way